### PR TITLE
Add grafana to openshift

### PR DESCRIPTION
**Pull request by sophokles73**
Build our own Grafana image from scratch which includes our Hono dashboard and datasource in the "template" DB at /tmp/grafana.db.

### CODE REVIEW

File: site/content/getting-started.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ In order to enable *Swarm Mode* on your *Docker Engine* run the following comman
 Please refer to the [Docker Swarm Mode documentation](https://docs.docker.com/engine/swarm/swarm-mode/) for details.
 
 {{% warning %}}
-You will need at least Docker version 1.13.0 in order to run the example in this guide. By the time of writing, the latest released version of Docker was 1.13.1.
+You will need at least Docker Engine version 1.13.1 in order to run the example in this guide. By the time of writing, the latest released version of Docker was 17.06.0.
```
Comment (ppatierno) [inline site/content/getting-started.md]: what's the problem with the 1.13.0 version ?

File: metrics/grafana/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    chown -R root:root /var/log/grafana && \
+    chmod +x /run.sh && \
+    chmod -R o+r /etc/grafana && \
+    chmod -R o+r /var/lib/grafana
+
+EXPOSE 3000
+
+VOLUME ["/var/lib/grafana", "/var/log/grafana", "/etc/grafana"]
+
+ENTRYPOINT ["/run.sh"]
```
Comment (ppatierno) [inline metrics/grafana/Dockerfile]: new line

File: metrics/grafana/run.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+fi
+
+exec /usr/sbin/grafana-server      \
+  --homepath=/usr/share/grafana                 \
+ --config=$CONFIG_FILE             \
+  cfg:default.log.mode="console"                \
+  cfg:default.paths.data="$GF_PATHS_DATA"       \
+  cfg:default.paths.logs="$GF_PATHS_LOGS"       \
+  cfg:default.paths.plugins="$GF_PATHS_PLUGINS" \
+"$@"
```
Comment (ppatierno) [inline metrics/grafana/run.sh]: new line

File: example/openshift/openshift_deploy.sh [hunk 4]
```diff
@@ -27,6 +27,11 @@ oc create -f $HONO_HOME/example/target/classes/META-INF/fabric8/openshift/hono-p
 # starting to deploy Eclipse Hono (developer user)
 oc login -u developer -n hono
 
+echo
```
Comment (ppatierno) [inline example/openshift/openshift_deploy.sh]: is it just a typo ?

File: site/content/getting-started.md (hunk 1 above)
Comment (sophokles73) [inline site/content/getting-started.md]: The docker compose file version 3.1 (which introduces support for secrets), is supported starting from 1.13.1.

File: example/openshift/openshift_deploy.sh (hunk 4 above)
Comment (sophokles73) [inline example/openshift/openshift_deploy.sh]: no, just wanted to output an empty line :-)

**Comment by sophokles73** [comment]
I think I would also like to remove the AWS credentials handling in the `run.sh` script. WDYT @ppatierno ?

**Comment by ppatierno** [comment]
Agree ! We don't need them.

**Pull request merged**